lib/rules/hidden-content.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "hidden-content",
+  "matches": "hidden-content.js",
+  "tags": [],
+  "metadata": {
+    "description": "Informs users about hidden content.",
+    "help": "Manually check elements that ar ehidden with display: none, visibility: hidden, or aria-hidden: \"true\""
+  },
+  "all": [],
+  "any": [
+    "hidden-content"
+  ],
+  "none": []
+}
```

Comment:
We should probably disable this rule by default in axe-core and enable it in the Attest extension.